cms/settings/base.py

```diff
@@ -219,8 +219,7 @@
 if "read_replica" not in DATABASES:
     DATABASES["read_replica"] = deepcopy(DATABASES["default"])
 
-# Ensure the correct connection is used depending on the query
-DATABASE_ROUTERS = ["cms.core.db_router.ReadReplicaRouter"]
+DATABASE_ROUTERS = ["cms.core.db_router.ExternalEnvRouter", "cms.core.db_router.ReadReplicaRouter"]
```

Comment:
Note for other devs: https://docs.djangoproject.com/en/5.1/topics/db/multi-db/#using-routers explains how this works

Reply:
Q: Would we want this change in base settings, or perhaps just in `production.py`? My worry is that added complication isn't really needed for dev or test environments.